clients/middleware/authorization.go

```diff
@@ -89,6 +89,10 @@ func (am *authorizationMiddleware) CreateClients(ctx context.Context, session au
 		}
 	}
 
+	if err := am.RoleManagerAuthorizationMiddleware.AuthorizeMembers(ctx, session, []string{session.UserID}); err != nil {
+		return []clients.Client{}, []roles.RoleProvision{}, errors.Wrap(svcerr.ErrAuthorization, err)
```

Comment:
@nyagamunene 
I think this is not needed, Because during the authorization the requested user will be check user have access to domain as member.